### PR TITLE
added profit-loss for control structure

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Rohan-Shakya** [comment]
Thank you Saljesh !! for your contribution.

File: control structure/profit-loss.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    // and then conditions will be applied accordingly
+
+    if(CP > SP){
+        loss = CP - SP;
+        printf("The loss is %f", loss);
+    }
+    else{
+        profit = SP - CP;
+        printf("The profit is %f", profit);
+    }
```
Comment (Rohan-Shakya) [inline control structure/profit-loss.c:23]: getch() ???

Comment (Rohan-Shakya) [reply control structure/profit-loss.c:23]: everything is goood !! Nice 👍👍👍

Comment (Rohan-Shakya) [inline control structure/profit-loss.c:23]: this '}' is  missing !!

**Review by Rohan-Shakya — APPROVED** [review]
